### PR TITLE
eliding second person in definitions

### PR DESCRIPTION
**Pull request by RacheltheEditor**
redo of #152

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1454,8 +1456,10 @@ Long-term capital gains taxes
 
 ### Federal taxes
 
-🄳 **Income tax** is the money paid by individuals to federal, state, and, in some cases,
-local governments, and includes taxation of ordinary income and capital gains.
+🄳 **Income tax** is the money paid by individuals (generally, U.S. citizens, residents, and
+some [foreigners](https://www.irs.gov/individuals/international-taxpayers/nra-withholding))
```
Comment (RacheltheEditor) [inline README.md]: @jlevy also wants to add this link for "[file](https://www.irs.gov/publications/p17#en_US_2017_publink1000170407) and pay"

Comment (jlevy) [reply README.md]: Move parenthetical to end sentence:

Generally, U.S. citizens, residents, and
some [foreigners](https://www.irs.gov/individuals/international-taxpayers/nra-withholding) [must file](https://www.irs.gov/publications/p17#en_US_2017_publink1000170407) and pay federal income tax.

Comment (RacheltheEditor) [reply README.md]: Fixed.

File: README.md [hunk 2]
```diff
@@ -440,7 +441,7 @@ issuing stock is how companies often choose to do this.
   **shareholders (or stockholders)**. Founders, investors, employees, board members,
   contractors, advisors, and other companies, like law firms, can all be shareholders.
 - 🄳 Stock ownership is often formalized on **stock certificates**, which are fancy pieces of
-  paper that prove you own the stock.
+  paper that prove who owns the stock.
```
Comment (jlevy) [inline README.md]: 👍

**Pull request merged**